suite-native/module-home/src/screens/HomeScreen/components/EmptyConnectedDeviceState.tsx

```diff
@@ -14,8 +14,9 @@ const cardStyle = prepareNativeStyle(utils => ({
     flex: 1,
     justifyContent: 'center',
     alignItems: 'center',
-    paddingVertical: utils.spacings.sp32,
-    paddingHorizontal: utils.spacings.sp24,
+    paddingTop: utils.spacings.sp32,
+    paddingBottom: utils.spacings.sp16,
+    paddingHorizontal: utils.spacings.sp16,
```

Comment:
I try to maintain this order – top, horizontal, bottom. Inspired by [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/padding):
```
/* top | left and right | bottom */
padding: 1em 2em 2em;
```

Feel free to ignore. 🙂